catkin_lintにPathの指定がない

<!-- 不具合のテンプレート -->
**概要**
下記のエラーを取得したので調査した。
https://github.com/sbgisen/outdoor_navigation_tools/runs/4439795966?check_suite_focus=true
```
catkin_lint: cannot load rosdep database: No module named 'rosdep2'
catkin_lint: unknown dependencies will be ignored
catkin_lint: no path given and no package.xml in current directory
Error: Process completed with exit code 66.
```
1. `catkin_lint: cannot load rosdep database: No module named 'rosdep2'`
   1. pip3 install rosdepが必要（[参考](https://demura.net/robot/16518.html) )
3. `catkin_lint: no path given and no package.xml in current directory`
   1. [公式](https://fkie.github.io/catkin_lint/usage/) にあるとおり、`catkin_lint **path**`で指定する必要がある

**再現手順**
手元で`catkin_lint`を実行

**修正しないとどう困るか**
catkin_lintが正しく使えない。

**原因**
下記でPathを指定していない。
https://github.com/sbgisen/.github/blob/7306baeed9873c69aebfce4a4d6a994c0eaaf1b7/.github/workflows/ros-test.yml#L122-L124

**修正案**
`catkin_lint  path`を実行するようにする。

--- a/.github/workflows/ros-test.yml
+++ b/.github/workflows/ros-test.yml
@@ -125,9 +125,9 @@ jobs:
         env:
           REVIEWDOG_GITHUB_API_TOKEN: ${{ github.token }}
         run: |
-          pip3 install catkin_lint
+          pip3 install catkin_lint rosdep
           source ../../devel/setup.bash
-          catkin_lint
+          catkin_lint ${{ github.workspace }}/ros/src/${{ github.event.repository.name }}
         shell: bash
         working-directory: ${{ github.workspace }}/ros/src/${{ github.event.repository.name }}
       - name: Download clang tidy config

--- a/.github/workflows/ros-test.yml
+++ b/.github/workflows/ros-test.yml
@@ -127,7 +127,7 @@ jobs:
         run: |
           pip3 install catkin_lint rosdep
           source ../../devel/setup.bash
-          catkin_lint ${{ github.workspace }}/ros/src/${{ github.event.repository.name }}
+          catkin_lint .
         shell: bash
         working-directory: ${{ github.workspace }}/ros/src/${{ github.event.repository.name }}
       - name: Download clang tidy config

